VeriD/PublicKeys/1.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<IssuerPublicKey xmlns="http://www.zurich.ibm.com/security/idemix">
+   <Counter>1</Counter>
```

Comment:
The first public key should have `<Counter>0</Counter>` and should be called 0.xml.